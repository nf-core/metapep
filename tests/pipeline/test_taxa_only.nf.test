nextflow_pipeline {

    name "Test Workflow main.nf"
    script "main.nf"
    tag "test_taxa_only"
    tag "pipeline"

    test("test taxa input") {

        when {
            params {
                outdir = "$outputDir"
            }
        }

        then {
            assert workflow.success
        }

    }

}
