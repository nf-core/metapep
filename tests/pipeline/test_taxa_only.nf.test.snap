{
    "entrez_data": {
        "content": [
            "microbiomes_entities.entrez.tsv:md5,d74ec125c02485cae58a08a334bf4844"
        ],
        "meta": {
            "nf-test": "0.8.4",
            "nextflow": "24.04.2"
        },
        "timestamp": "2024-06-26T08:14:54+0000"
    },
    "db_tables": {
        "content": [
            "alleles.tsv:md5,77994270b1eab3e8363c90d8d09780fa",
            "conditions_alleles.tsv:md5,9db7e0afc739b23c430877e336b0143f",
            "conditions.tsv:md5,a6fe5daaf11a90b61c83c9934969c52c",
            "entities_proteins.tsv:md5,182d6c0cec8aeb466348d9df96c279b0",
            "entities.tsv:md5,c766a12bbb1d4ef4707be7077f774191",
            "microbiomes_entities.no_weights.tsv:md5,2d18c7210b304eb7593106073bc3686f",
            "microbiomes_entities.nucl.tsv:md5,84b6b907695caaddc959682504749832",
            "microbiomes_entities.tsv:md5,9fc7c15404073c65179dd908dc9fde78",
            "microbiomes.tsv:md5,05bafa72c953e6cb0bd72c296d6fe707",
            "proteins_peptides.tsv:md5,3cc6b7bdc6b91016cce5a94ff4a0fa6e",
            "peptides.tsv.gz:md5,0e3c1e87feb2aac34de1ba3b246f27ca",
            "proteins.tsv.gz:md5,e51aef45c63fd5047e8d21eab26038a5",
            "predictions.tsv.gz:md5,6a931de30c25a2d984193d2586a7eb4b",
            "stats.tsv:md5,6a372e5230fdd274674276300838dcfa"
        ],
        "meta": {
            "nf-test": "0.8.4",
            "nextflow": "24.04.3"
        },
        "timestamp": "2024-09-13T18:46:03.686298061"
    },
    "logs": {
        "content": [
            "prediction_warnings.log:md5,d41d8cd98f00b204e9800998ecf8427e",
            "SUCCEEDED_unify_peptide_lengths.log:md5,c27fb56e478b975c57c910011935dd55",
            "download_proteins.log:md5,488e1962cf079d466188481d526495f2"
        ],
        "meta": {
            "nf-test": "0.8.4",
            "nextflow": "24.04.3"
        },
        "timestamp": "2024-07-18T14:49:36.871618584"
    }
}